pkg/asset/internal/templates.go

```diff
@@ -349,20 +359,25 @@ spec:
       - name: healthz
         image: gcr.io/google_containers/exechealthz-amd64:1.1
         resources:
+          # keep request = limit to keep this container in guaranteed class
           limits:
             cpu: 10m
```

Comment:
nit: this is an slightly old version of kube-dns. A recent commit during beta removed a few of the cpu limits for kube-dns. See: https://github.com/kubernetes/kubernetes/commit/2b805e3d0f4bf9d377325e0433c799c3f4a56fb2